projects/tfb-presentation.md

```diff
@@ -0,0 +1,65 @@
+3. Show the evolution of your project:
+   - Pen and paper --> Figma prototype --> MVP
+4. Consider showing DB schema, flow diagrams, other graphics if they add to the story you're telling
+5. Weave standout quotes from user research/testing into the story
+6. Talk about your processes, including:
+   - How you structured your work (project kanban board)
+   - User story estimation, and how this compared with actual time taken
+   - How you stayed on track (stand ups, retros, sprint reviews, planning)
+7. Show your tech stack:
+   - Which technologies you used (logos are handy for showing this)
+   - Why you opted to use those ones vs others
+8. GIF screengrabs are great for showing how UI components work (Have a look at Loom)
+9. Data viz (i.e. stacked bar charts, burndown charts) is a great way to show estimated vs actual velocity
+10. Make sure code snippets really are **snippets**! (viz. keep them short)
+11. Mention any interesting work you'd tackle if you had more time
+12. Have a ‘Thank you/Questions?’ slide so that your audience knows when to clap at the end of your presentation!
+
+### Structure tips
+
+1. Introduce yourselves and your role in the project
+2. Give a brief overview of what your presentation will cover at the start
+3. Use the rule of three to highlight key points on slides
+4. Have some kind of progress indicator on your slides - helps your audience to keep track of what you're talking about
+
+### Delivery tips
+
+1. Spread talking time equally between you
+2. It looks slick when everyone knows who’s delivering each part of the presentation. Try to work this out beforehand
+3. Designate someone to change slides
+4. Talk about team achievements first, individual achievements last...
+5. ...but make sure everyone has a chance to talk about an individual contribution that they are proud of. This could be phrased as a challenge related to your role that you overcame.
+6. Explaining code that someone else wrote is a good way to credit your teammates while showing off your own understanding of the codebase
+7. Refer to line numbers to make it clear which part of the code you're talking about
+8. Practice your presentation at least once together
+
+---
+
+### Example slide decks
+
+- [Team LsX Partners Day Canva slides](https://www.canva.com/design/DADdwXpAcTM/JYHCDzCNMjGMf84AQ9W2Kw/view?utm_content=DADdwXpAcTM&utm_campaign=designshare&utm_medium=link&utm_source=sharebutton)
+- [Team SquareCircle Partners Day slides](https://www.canva.com/design/DADdxRwH7ZM/_wGKHjqdVWecQYmtlD9pig/view)
+
+---
+
+### Handy Tools
+
+[Excalidraw](https://excalidraw.com/) - make "handdrawn" mock ups of your UI
+[Creately](https://creately.com/) - create process diagrams (i.e. to map the flow of your app)
+[Dbdiagram.io](https://dbdiagram.io/home) - visualise your db schema
+[Gyazo](https://gyazo.com/captures) - record GIFs of your screen
+[Canva](https://www.canva.com/) - professional presentation templates
```

Comment:
These either need to be a list or have an additional line break (so the markdown engine puts each in a new paragraph). Right now they are all output on a single line in the preview

Reply:
Good spot, I copied from a HackMD 😅